src/app/product-list/product/product.component.html

```diff
@@ -0,0 +1,16 @@
+<section class="container-fluid">
+    <section class="row">
+        <div class="card col-md-3 col-sm-6 col-12 p-1" *ngFor="let product of displayProducts">
+            <div class="card-body">
+                <p class="card-title">{{product.name}}</p>
+                <img class="card-img-top" src="assets/{{product.imageURL}}" alt="{{product.name}}"
+                    title="{{product.name}}">
+            </div>
+            <p class="card-text">{{product.description}}</p>
```

Comment:
Use ellipsis `...` when the line count exceeds the given limit

Reply:
done